rules/windows/builtin/win_exploit_cve_2021_1675_printspooler.yml

```diff
@@ -0,0 +1,38 @@
+title: Possible CVE-2021-1675 Print Spooler Exploitation
+id: 4e64668a-4da1-49f5-a8df-9e2d5b866718
+description: Detects events of driver load errors in print service logs that could be a sign of successful exploitation attempts of print spooler vulnerability CVE-2021-1675
+author: Florian Roth, KevTheHermit, fuzzyf10w
+status: experimental
+level: high
+references:
+    - https://github.com/hhlxf/PrintNightmare
+    - https://github.com/afwu/PrintNightmare
+    - https://twitter.com/fuzzyf10w/status/1410202370835898371
+date: 2021/06/30
+tags:
+    - attack.execution
+    - cve.2021-1675
+logsource:
+    product: windows
+    service: printservice-admin
+detection:
+    selection:
+        EventID: 
+            - 808   # old id
+            - 4909  # new id
```

Comment:
@Neo23x0, Under which case is this new EventID generated? I am currently only observing 808.